### DISCUSSION TITLE
Output image return only one class

### DISCUSSION THREAD

**Discussion by papabricio** [opengeos/segment-geospatial#102] [Q&A] [answered]
Even with the unique parameter set True when I run the generate function the output image returns only with a single class of value 255. Here is the input image and then the output mask:
![image](https://github.com/opengeos/segment-geospatial/assets/63574691/e8e426ff-8afb-4331-b33e-635b901eda5d)
![image](https://github.com/opengeos/segment-geospatial/assets/63574691/514aa100-9a98-498d-82ce-45160a66d8ac)

I used the following example, however using my own image
https://samgeo.gishub.org/examples/automatic_mask_generator/

**Comment by giswqs** [comment, marked as answer]
Try out the example without making any changes. You should see the random colored result. 
https://samgeo.gishub.org/examples/automatic_mask_generator/

**Reply by papabricio** [reply]
Yes, with the coordinates provide in the example I have the expected results. But if I change the dataset to my own or even if I only change the bbox in the tms_to_geotiff function to my area, the generate method return only with one category.
